### ISSUE TITLE
Token code not zero-padded to correct length

### ISSUE THREAD

**Issue by conormkelly** [xarenard/simpleotp#1]
https://github.com/xarenard/simpleotp/blob/c974078900aeac35676acf9448bef1ce1f5f0e64/lib/otp.js#L76

Hi @xarenard ,

I noticed that tokens that begin with a leading `0` are sometimes not the correct length.

The problem is that the if statement in the referenced line will only add _one_ zero.
If there are _two_ or more leading zeros required, the generated code is not the correct length.

```
// Sample rfc_6238 test data - should be of length "8"

const testDatas = [
  // ...
  { seconds: 3060, algorithm: 'sha1', expected_token: '00629694' },
  { seconds: 4620, algorithm: 'sha256', expected_token: '00836417' },
  { seconds: 5100, algorithm: 'sha512', expected_token: '00458766' }
]
```

I can submit a PR if you'd like, but it should just be a matter of updating this `if` statement to a `while` loop.

**Comment by xarenard** [comment]
Hi,

Thanks for the feedback.I will have a look. Indeed no need for a PR

Cheers

Xavier

**Comment by xarenard** [comment]
An updated version has been published (version 0.7.7).Let me know if it does solve your issue.

**Comment by conormkelly** [comment]
Yes - looks good. Thank you for resolving this so quickly. Much appreciated!
